core/data/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/data/remoteconfig/DefaultRemoteConfigApi.kt

```diff
@@ -0,0 +1,26 @@
+package io.github.droidkaigi.confsched2023.data.remoteconfig
+
+import dev.gitlive.firebase.Firebase
+import dev.gitlive.firebase.remoteconfig.get
+import dev.gitlive.firebase.remoteconfig.remoteConfig
+
+class DefaultRemoteConfigApi : RemoteConfigApi {
+
+    private val firebaseRemoteConfig = Firebase.remoteConfig
+
+    /**
+     * If you want to change the interval time to fetch, please change it here
+     */
+//    init {
+//        CoroutineScope(Dispatchers.IO).launch {
+//            firebaseRemoteConfig.settings {
+//                minimumFetchIntervalInSeconds = 12 * 3600L
+//            }
+//        }
+//    }
```

Comment:
The original Firebase library has an implementation that automatically fetches changes when a value is updated, but the library for KMM does not, so the only way is to set `minimumFetchIntervalInSeconds` to fetch the value periodically.